docs/DESIGN.md

```diff
@@ -401,10 +401,18 @@ ACME v1/v2:
 
 ```
 1: Client ---cert--> WFE/WFEv2
-2:                   WFE/WFEv2 ---RevokeCertificateWithReg--> RA
-3:                   WFE/WFEv2 <------------return----------- RA
+2:                   WFE/WFEv2 ---RevokeCertByApplicant--> RA
+3:                   WFE/WFEv2 <-----------return--------- RA
```

Comment:
Nit, maybe time to remove WFE (v1) from this as well?

```suggestion
2:                   WFEv2 ---RevokeCertByApplicant--> RA
3:                   WFEv2 <-----------return--------- RA
```

Reply:
Yeah, there's a lot of other stuff to clean up in the other diagrams in this doc, but I've taken care of at least these ones now.